internal/indexer/doctomarkdown.go

```diff
@@ -34,12 +36,25 @@ func doctomarkdown(ctx context.Context, r io.Reader, matchingTags []protocol.Doc
 	return buf.String(), nil
 }
 
+type DocsRefDefInfo struct {
+	Document string
+	Ranges   []reader.Range
+}
+
 type converter struct {
```

Comment:
Since this whole file is used only in tests, can we give it a `_test.go` suffix?